pkg/reconciler/events/cloudevent/cloudeventclient.go

```diff
@@ -35,11 +36,13 @@ type CECKey struct{}
 func withCloudEventClient(ctx context.Context, cfg *rest.Config) context.Context {
 	logger := logging.FromContext(ctx)
 
-	p, err := cloudevents.NewHTTP()
+	var useOnceTransport http.RoundTripper = &http.Transport{
```

Comment:
Can you add a comment describing why we're doing this, with a link to the bug and a TODO to remove it when some upstream bug is closed? I don't want to forget we're doing this.

Reply:
Good point, I will do.
I don't have an upstream bug created yet, but I'll link the pipeline bug for context.